### DISCUSSION TITLE
Portuguese Translation

### DISCUSSION THREAD

**Discussion by nazarepiedady** [vuejs-translations/guidelines#26] [Q&A]
# **The Portuguese translation is 99% concluded**.

## Purpose

The purpose of this discussion is to track the document translation process 
to Portuguese (more accessible to all speakers, being natives or not).

## How to participate

If you are a person interested in being part of translating process, you should 
start making a fork of the documentation repository that is in my GitHub profile, 
then you should comment here and issue there about the parts or sections of the 
documentation that you are translating.

You can participate in the project here: https://github.com/nazarepiedady/vue3-docs-pt

## About me

I am a Software Programmer based in Angola (the second major Portuguese-speaking country), I am also the person behind the following open-source contribution: [Nuxt 2 Official Portuguese Documentation](https://nuxtjs.org/pt/), [Vite Official Portuguese Documentation](https://github.com/vitejs/docs-pt), [Vitest Portuguese Documentation](https://github.com/nazarepiedady/vitest-docs-pt), [Sass Portuguese Documentation](https://github.com/nazarepiedady/sass-docs-pt), [Vue Router Portuguese Documentation](https://github.com/nazarepiedady/vue-router-docs-pt), [Pinia Portuguese Documentation](https://github.com/nazarepiedady/pinia-docs-pt) and more.

**Comment by nazarepiedady** [comment]
I am sorry to bother you fellows, I understand that you are busy so I mentioned
all I could to see whether someone could attend to my tread here:

@hootlex @danielroe @danielkellyio @antfu @lmiller1990 @NataliaTepluhina
@yyx990803 @posva @shentao @sdras @pikax @LinusBorg @nickmessing
@kiaking @kazupon @Jinjiang @wxsms @veaba @jay-es

**Comment by yrpcfcp** [comment]
Hey

I would like to contribute to the translation. How can I help?

**Reply by nazarepiedady** [reply]
@yrpcfcp Currently I am waiting for the repository of the translation to be added to the
Vue.js Translations organisation so until it happens you can follow the project through my 
GitHub profile.

**Comment by viniciusdeliz** [comment]
This project is feeling stuck. I definitely miss the workflow and community activity we had with the former Vue 3 documentation translation under the [vuejs-br/docs-next](https://github.com/vuejs-br/docs-next) repo. What can we do to improve this scenario?

**Reply by nazarepiedady** [reply]
@viniciusdeliz you feel this because you are not following the repository and
its website to see how much content is already available to be consumed.

I do not know how a project that has its latest commit done 3 weeks ago is stuck,
you should understand that the maintainer of the project is working on others
projects too as shown above.

I already explained to you what kind of help would be great, because I would not
have too much time to review the pull request because most of the time I am
doing the work offline and then when I conclude some I push it to the repository.

**Reply by nazarepiedady** [reply]
@viniciusdeliz I understand and I know the people behind the [vuejs-br](https://github.com/vuejs-br/),
because I am part of Vue.js Brasil, Vue.js Portugal, and Vue.js Angola so the people there know
about this project, I already explained when the project will be open to receive pull requests,
so until the project concludes its first version then contributions will be welcomed.

**Comment by nazarepiedady** [comment]
I think it is time to add [nazarepiedady/vue3-docs-pt](https://github.com/nazarepiedady/vue3-docs-pt) to [vuejs-translations](https://github.com/vuejs-translations).

I am mentioning you because I would like to know which one of you I need to talk 
to transfer the [nazarepiedady/vue3-docs-pt](https://github.com/nazarepiedady/vue3-docs-pt) to [vuejs-translations](https://github.com/vuejs-translations).

@kazupon @Jinjiang @kiaking @wxsms @veaba 
@ShenQingchuan @jay-es @mahmudunnabikajal @nazar1ua.

**Reply by Jinjiang** [reply]
Hi, @nazarepiedady great job! I'm glad to help you to achieve that. I will send you an invitation to the org. After accepting that you should be free to transfer the repo and maintain it in this org.

**Reply by nazarepiedady** [reply]
@jinjiang, thanks for your help. I already accepted the invitation and also already transferred the repository to this organization with the proper rename to match the pattern used which is `docs-pt`.

**Reply by viniciusdeliz** [reply]
@Jinjiang is there any estimate of when the Portuguese documentation will be published in the [Vue official docs](https://vuejs.org/)?

**Reply by nazarepiedady** [reply]
@viniciusdeliz Today the people in the Vue team are too busy, the best way to contact one of them is by using Discord.
I will ask the same question on Discord to @Jinjiang.

**Reply by Jinjiang** [reply]
@nazarepiedady @viniciusdeliz I can hear you. No worries. :-)

Just would like to double-check whether the translation is totally finished. Because the official publishment and announcement will be made once it's done (as step 5 says).

Thanks.

**Reply by Jinjiang** [reply]
Update: just had a quick chat with @nazarepiedady and I prepared this PR to English docs
https://github.com/vuejs/docs/pull/2448
The else part would be done by @yyx990803 🙏

**Comment by MayaraRMA** [comment]
Hey, do you think is there any difference for pt-br? if not, I would like to help if you still need some contribution.

**Reply by nazarepiedady** [reply]
@MayaraRMA there is a little difference from the pt-br translation, but this will be changed soon in order to unify all Portuguese variations.

In fact, this project achieved its goal with the help of Brazilian programmers. But at this moment the project is not receiving contributions in pull request format anymore because we finished the translations of the content and are in the phase of reviewing the content to match more with all Portuguese variations.

If you are interested in contributing, you can do with issues pointing out some usage of words that are difficult to understand or are wrong.

Go to [vuejs-translations/docs-pt](https://github.com/vuejs-translations/docs-pt)
